app/commands/solution/search_community_solutions.rb

```diff
@@ -0,0 +1,26 @@
+class Solution
+  class SearchCommunitySolutions
+    include Mandate
+
+    DEFAULT_PAGE = 1
+    DEFAULT_PER = 25
```

Comment:
Was there a reason not to make these private?

Reply:
Now: I copied this from the Searching of User Solutions.
Previously: 🤷 No, probably not!